R/derive_param_framingham.R

```diff
@@ -0,0 +1,355 @@
+#' Adds a Parameter for Framingham Heart Study Cardiovascular Disease
+#' 10-Year Risk Score
+#'
+#' Adds a record for framingham score (FCVD101) for each by group
+#' (e.g., subject and visit) where the source parameters are available.
+#'
+#' @param dataset Input dataset
+#'
+#'   The variables specified by the `by_vars` parameter, `PARAMCD`, and
+#'   `AVAL` are expected.
+#'
+#'   The variable specified by `by_vars` and `PARAMCD` must be a unique key of
+#'   the input dataset after restricting it by the filter condition (`filter`
+#'   parameter) and to the parameters specified by `sysbp_code`, `chol_code`
+#'   and `hdl_code`.
+#'
+#' @param sysbp_code Systolic blood pressure parameter code
+#'
+#'   The observations where `PARAMCD` equals the specified value are considered
+#'   as the systolic blood pressure assessments.
+#'   Permitted Values: character value
+#'
+#' @param chol_code Total serum cholesterol code
+#'
+#'   The observations where `PARAMCD` equals the specified value are considered
+#'   as the total cholesterol assessments. This must be measured in mg/dL
+#'   Permitted Values: character value
+#'
+#' @param cholhdl_code HDL serum cholesterol code
+#'
+#'   The observations where `PARAMCD` equals the specified value are considered
+#'   as the HDL cholesterol assessments. This must be measured in mg/dL
+#'   Permitted Values: character value
+#'
+#' @param age Subject age
+#'
+#'   A variable containing the subject's age
+#'   Permitted Values: Numeric value
+#'
+#' @param sex Subject sex
+#'
+#'   A variable containing the subject's sex
+#'   Permitted Values: M, F
+#'
+#' @param smokefl Smoking status flag
+#'
+#'   A flag indicating smoking status: Y=current smoker, N=otherwise
+#'   Permitted Values: Y, N
+#'
+#' @param diabetfl Diabetic flag
+#'
+#'   A flag indicating diabeti status: Y=diabetic, N=otherwise
+#'   Permitted Values: Y, N
+#'
+#' @param trthypfl Treated with hypertension medication flag
+#'
+#'   A flag indicating if a subject was treated with hypertension medication:
+#'   Y=treated for high blood pressure, N=Not treated for high blood pressure
+#'   Permitted Values: Y, N
+#'
+#' @inheritParams derive_derived_param
```

Comment:
Typo?

Reply:
Hey @bms63, would the typo be replacing the words `high blood pressure` with `hypertension`?